packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -263,7 +263,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
             onClick={this._onPanelClick}
             ariaLabel={closeButtonAriaLabel}
             data-is-visible={true}
-            iconProps={{ iconName: 'Cancel' }}
+            iconProps={{ iconName: 'Cancel', ariaLabel: closeButtonAriaLabel }}
```

Comment:
When I test this locally, I don't see any difference in screen reader behavior by adding the ariaLabel to the icon, since it's already attached to the button. Are there specific scenarios I should test?

Reply:
This seems really fishy!

The button should be the thing that reads closeButtonAriaLabel, not the Icon itself. No?

@jspurlin or @cschlechty Please tell me if i'm wrong.

Reply:
If a button with label isn't being read, we should file a bug with narrator. The icon should be role="presentation" and not hidden. A label there shouldn't be necessary.

Reply:
The Button has an ariaLabel (line: 264) and I'm having it read out just fine with narrator. It doesn't make sense for this icon to have a label